src/Plugin.php

```diff
@@ -45,6 +45,9 @@ public function onLoad()
         add_action('wp_get_attachment_image_src', [$attachment, 'attachmentImageSrc'], 99, 4);
         add_filter('media_view_strings', [$attachment, 'mediaStrings']);
 
+        remove_filter('the_content', 'wp_make_content_images_responsive');
+        add_filter('the_content', [$attachment, 'makeContentImagesResponsive']);
```

Comment:
I'm aware this is far from ideal as other plugins could be doing the same thing but there's not really any other way to add support for `srcset`. Without it this plugin would cause original image sizes to be served to all browsers which I consider unacceptable.